--- DetailView.tpl	2010-06-23 16:00:26.000000000 +0200
+++ DetailView.tpl.flac	2010-04-03 03:55:38.000000000 +0200
@@ -70,6 +70,11 @@
 		{{assign var='columnsInRow' value=$rowData|@count}}
 		{{assign var='columnsUsed' value=0}}
 	    {{foreach name=colIteration from=$rowData key=col item=colData}}
+
+
+		{{if $colData.field.name != ''}}
+		{if $bean->field_defs.{{$colData.field.name}}.acl < 1}
+		{{/if}}
 			<td width='{{$def.templateMeta.widths[$smarty.foreach.colIteration.index].label}}%' scope="row">
 				{{if isset($colData.field.customLabel)}}
 			       {{$colData.field.customLabel}}
@@ -111,6 +116,14 @@
 					{{sugar_field parentFieldArray='fields' vardef=$fields[$colData.field.name] displayType='detailView' displayParams=$colData.field.displayParams typeOverride=$colData.field.type}}
 				{{/if}}
 			</td>
+
+		{{if $colData.field.name != ''}}
+		{else}
+			<td class='tabDetailViewDL'>&nbsp;</td><td class='tabDetailViewDF'>&nbsp;</td>
+		{/if}
+		{{/if}}
+
+
 		{{/foreach}}
 	</tr>
 	{{/foreach}}
